system/webrtc/device/video.py

```diff
@@ -32,10 +33,12 @@ async def recv(self):
 
     packet = av.Packet(evta.header + evta.data)
     packet.time_base = self._time_base
-    packet.pts = self._pts
 
-    self.log_debug("track sending frame %s", self._pts)
-    self._pts += self._dt * self._clock_rate
+    if getattr(self,"_t0_ns", None) is None:
```

Comment:
don't do this, just set a default in the constructor. in general, avoid getattr unless absolutely necessary